### PR TITLE
Add two RuntimeError exceptions: add the URL of a failed download.

### PR DESCRIPTION
**Pull request by mykaul**
1. Of a template.
2. Of a repo metadata file.
I got both on ovirtmaster testing right now. 

For example:
```
  # Create disks for VM lago_basic_suite_master_engine: ERROR (in 0:04:14)
@ Initialize and populate prefix: ERROR (in 0:04:15)
Error occured, aborting
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/lago/cmd.py", line 447, in main
    cli_plugins[args.verb].do_run(args)
  File "/usr/lib/python2.7/site-packages/lago/plugins/cli.py", line 180, in do_run
    self._do_run(**vars(args))
  File "/usr/lib/python2.7/site-packages/lago/log_utils.py", line 598, in wrapper
    func(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/lago/cmd.py", line 113, in do_init
    prefix.virt_conf(virt_conf, repo, store)
  File "/usr/lib/python2.7/site-packages/lago/__init__.py", line 739, in virt_conf
    template_store,
  File "/usr/lib/python2.7/site-packages/lago/__init__.py", line 485, in _create_disk
    template_store.download(template_version)
  File "/usr/lib/python2.7/site-packages/lago/templates.py", line 646, in download
    temp_ver.download(temp_dest)
  File "/usr/lib/python2.7/site-packages/lago/templates.py", line 518, in download
    self._source.download_image(self._handle, destination)
  File "/usr/lib/python2.7/site-packages/lago/templates.py", line 203, in download_image
    self.open_url(url=handle, dest=dest)
  File "/usr/lib/python2.7/site-packages/lago/templates.py", line 160, in open_url
    raise RuntimeError('Failed to open URL %s' % (full_url))
RuntimeError: Failed to open URL http://10.35.18.63/repo/centos7/engine/v5.qcow2
```

### CODE REVIEW

File: contrib/ovirt/lib/ovirtlago/repoverify.py [hunk 1]
```diff
@@ -107,7 +107,10 @@ def fetch_xml(url):
     Returns:
         lxml.etree._Element: Root of the xml tree for the retrieved resource
     """
-    content = urllib2.urlopen(url).read()
+    try:
+        content = urllib2.urlopen(url).read()
+    except:
+        raise RuntimeError('Failed to open URL %s' % url)
```
Comment (david-caro) [inline contrib/ovirt/lib/ovirtlago/repoverify.py]: I don't thin we should swallow the exception, probably you can add the original exception message in the RuntimeError you are throwing so you have both, the url that failed, and the reason why

**Comment by mykaul** [comment]
@david-caro  - any idea what the above error is about ?

**Comment by david-caro** [comment]
The repos seem to be unreachable:

```
DEBUG util.py:399:  http://proxy.phx.ovirt.org:5000/fedora-updates/22/x86_64/repodata/repomd.xml: [Errno 14] HTTP Error 502 - Bad Gateway
```

**Comment by david-caro** [comment]
There's a bit of a mess with the commits still :/

**Comment by mykaul** [comment]
ARGH! I actually thought it looks good - I've just continued adding more fixes on top of this, without merging / rebasing any further. Will look into it...

**Comment by mykaul** [comment]
#141 is supposed to be a clean version. Hopefully.

**Pull request closed without merging**